internal/api/cluster.go

```diff
@@ -301,9 +326,9 @@ func handleUpdateCluster(c *Context, w http.ResponseWriter, r *http.Request) {
 	outputJSON(c, w, cluster)
 }
 
-// handleUpgradeCluster responds to PUT /api/cluster/{cluster}/kubernetes/{version},
+// handleUpgradeKubernetes responds to PUT /api/cluster/{cluster}/kubernetes/{version},
 // upgrading the cluster to the given Kubernetes version.
-func handleUpgradeCluster(c *Context, w http.ResponseWriter, r *http.Request) {
+func handleUpgradeKubernetes(c *Context, w http.ResponseWriter, r *http.Request) {
```

Comment:
Is this change OK? I think it makes the meaning of the method clearer.

Reply:
👍